### PR TITLE
Spanish translation: All short text corrected

### PR DESCRIPTION
**Pull request by hiperiondev**

### CODE REVIEW

File: Translations/es.po [hunk 1]
```diff
@@ -5694,337 +5599,283 @@ msgstr "De Murphy"
...
-#, fuzzy
 msgid "Small Chest"
-msgstr "Pecho pequeño"
+msgstr "Cofre Pequeño"
 
 #: Source/objects.cpp:5456
-#, fuzzy
 msgid "Chest"
-msgstr "Pecho"
+msgstr "Vofre"
```
Comment (NikoVP) [inline Translations/es.po:5722]:
```suggestion
msgstr "Cofre"
```

File: Translations/es.po [hunk 2]
```diff
@@ -5694,337 +5599,283 @@ msgstr "De Murphy"
...
 #: Source/objects.cpp:5480
-#, fuzzy
 msgid "Barrel"
 msgstr "Barril"
 
 #. TRANSLATORS: {:s} will be a name from the Shrine block above
 #: Source/objects.cpp:5484
-#, fuzzy
 msgid "{:s} Shrine"
 msgstr "{:s} Santuario"
```
Comment (NikoVP) [inline Translations/es.po:5755]:
```suggestion
msgstr " Santuario de {:s}"
```

File: Translations/es.po [hunk 3]
```diff
@@ -5694,337 +5599,283 @@ msgstr "De Murphy"
...
 msgid "Decapitated Body"
-msgstr "Cuerpo decapitado"
+msgstr "Cuerpo Decapitado"
 
+# o "de los ciegos"?
 #: Source/objects.cpp:5500
-#, fuzzy
 msgid "Book of the Blind"
-msgstr "Libro de los ciegos"
+msgstr "Libro del Ciego"
```
Comment (NikoVP) [inline Translations/es.po:5776]:
```suggestion
msgstr "Libro de los Ciegos"
```

File: Translations/es.po [hunk 4]
```diff
@@ -5694,337 +5599,283 @@ msgstr "De Murphy"
...
-#, fuzzy
 msgid "Book of Blood"
-msgstr "Libro de sangre"
+msgstr "Libro de Sangre"
 
+# o "resorte" ?
 #: Source/objects.cpp:5506
 #, fuzzy
 msgid "Purifying Spring"
 msgstr "Primavera Purificante"
```
Comment (NikoVP) [inline Translations/es.po:5786]:
```suggestion
msgstr "Manantial Purificante"
```

File: Translations/es.po [hunk 5]
```diff
@@ -5694,337 +5599,283 @@ msgstr "De Murphy"
...
-#, fuzzy
 msgid "Weapon Rack"
-msgstr "Estante de armas"
+msgstr "Estante de Armas"
 
 #: Source/objects.cpp:5516
-#, fuzzy
 msgid "Goat Shrine"
-msgstr "Santuario de la cabra"
+msgstr "Santuario de la Cabra"
```
Comment (NikoVP) [inline Translations/es.po:5794]:
```suggestion
msgstr "Santuario de las Cabras"
```

File: Translations/es.po [hunk 6]
```diff
@@ -5694,337 +5599,283 @@ msgstr "De Murphy"
...
-#, fuzzy
 msgid "Mushroom Patch"
-msgstr "Parche de hongos"
+msgstr "Parche de Hongos"
 
+# no se como traducir exactamente esto
 #: Source/objects.cpp:5543
 #, fuzzy
 msgid "Vile Stand"
 msgstr "Stand de Villa"
```
Comment (NikoVP) [inline Translations/es.po:5826]:
```suggestion
msgstr "Vil Estante"
```

File: Translations/es.po [hunk 7]
```diff
@@ -5694,337 +5599,283 @@ msgstr "De Murphy"
...
-#, fuzzy
 msgid "Slain Hero"
-msgstr "Héroe asesinado"
+msgstr "Héroe Asesinado"
 
 #. TRANSLATORS: {:s} will either be a chest or a door
 #: Source/objects.cpp:5553
-#, fuzzy
 msgid "Trapped {:s}"
 msgstr "Atrapado {:s}"
```
Comment (NikoVP) [inline Translations/es.po:5835]:
```suggestion
msgstr "{:s} Atrapado"
```

File: Translations/es.po [hunk 8]
```diff
@@ -6036,466 +5887,389 @@ msgstr ""
...
 #: Source/stores.cpp:1062 Source/stores.cpp:1249 Source/stores.cpp:1261
 #: Source/stores.cpp:1273
-#, fuzzy
 msgid "Would you like to:"
 msgstr "Le gustaría:"
 
 #: Source/stores.cpp:245
-#, fuzzy
 msgid "Talk to Griswold"
 msgstr "Habla con Griswold"
```
Comment (NikoVP) [inline Translations/es.po:6245]:
```suggestion
msgstr "Hablar con Griswold"
```

File: Translations/es.po [hunk 9]
```diff
@@ -6508,274 +6282,224 @@ msgstr "Atrás"
...
 #: Source/stores.cpp:614
-#, fuzzy
 msgid "Witch's shack"
-msgstr "Choza de la bruja"
+msgstr "Choza de la Bruja"
 
 #: Source/stores.cpp:616
-#, fuzzy
 msgid "Talk to Adria"
 msgstr "Habla con Adria"
```
Comment (NikoVP) [inline Translations/es.po:6314]:
```suggestion
msgstr "Hablar con Adria"
```

File: Translations/es.po [hunk 10]
```diff
@@ -6508,274 +6282,224 @@ msgstr "Atrás"
...
 #: Source/stores.cpp:943 Source/towners.cpp:160
-#, fuzzy
 msgid "Wirt the Peg-legged boy"
-msgstr "Wirt el chico de patas de palo"
+msgstr "Wirt el chico de la Pata de Palo"
 
 #: Source/stores.cpp:946 Source/stores.cpp:953
-#, fuzzy
 msgid "Talk to Wirt"
 msgstr "Habla con Wirt"
```
Comment (NikoVP) [inline Translations/es.po:6376]:
```suggestion
msgstr "Hablar con Wirt"
```

File: Translations/es.po [hunk 11]
```diff
@@ -6508,274 +6282,224 @@ msgstr "Atrás"
...
 #: Source/stores.cpp:1061
-#, fuzzy
 msgid "The Town Elder"
-msgstr "El anciano de la ciudad"
+msgstr "El Anciano del Pueblo"
 
 #: Source/stores.cpp:1063
-#, fuzzy
 msgid "Talk to Cain"
 msgstr "Habla con Cain"
```
Comment (NikoVP) [inline Translations/es.po:6428]:
```suggestion
msgstr "Hablar con Cain"
```

File: Translations/es.po [hunk 12]
```diff
@@ -10935,148 +10659,124 @@ msgstr ""
...
-#, fuzzy
 msgid "Lester the farmer"
 msgstr "Lester el granjero"
 
+# nut tiene muchos significados. Falta contexto
 #: Source/towners.cpp:260
 #, fuzzy
 msgid "Complete Nut"
-msgstr "Tuerca completa"
+msgstr "Tuerca Completa"
```
Comment (NikoVP) [inline Translations/es.po:10719]:
```suggestion
msgstr "Chiflado Completo"
```

File: Translations/es.po [hunk 13]
```diff
@@ -5104,588 +5102,495 @@ msgid "Bilefroth the Pit Master"
...
 #: Source/monster.cpp:4789
-#, fuzzy
 msgid "Total kills: {:d}"
 msgstr "Muertes totales: {:d}"
 
 #: Source/monster.cpp:4822
-#, fuzzy
 msgid "Hit Points: {:d}-{:d}"
-msgstr "Puntos de vida: {:d}- {:d}"
+msgstr "Puntos de Golpe: {:d}- {:d}"
```
Comment (NikoVP) [inline Translations/es.po:5379]:
```suggestion
msgstr "Vida: {:d}- {:d}"
```

**Review by NikoVP — CHANGES_REQUESTED** [review]
Hola, no soy nativo así no que puedo échate una mano directamente. Aún quedan muchísimas cosas a revisar, varios términos siguen mal traducidos tras la revisión. Te recomiendo usar Wikipedia y otras fuentes para comprobar que realmente significan las palabras y echar vistazo a otras traducciones. También te puedes unir al canal de DeviutionX en Discord. Hay otra persona que empezó la traducción en español y no la termino, quizás la puedes usar como referencia.  Ven al canal y podamos charlar y resolver algunas dudas que tienes. Un saludo!

**Comment by hiperiondev** [comment]
Gracias por ayudar.
En la traducción voy poniendo notas donde no estoy seguro. En muchos casos es dificil sin contexto.
Estoy terminando de revisar los dialogos que son bastante largos.
Es complicado para hacerlo solo y sin un feedback.
Cuando complete los dialogosvoy a ir a Discord a ver si se suma alguien más

**Comment by hiperiondev** [comment]
Tambien voy a revisar la traducción en Italiano, ya que es mi segunda lengua. No se en que estado está

**Comment by NikoVP** [comment]
Entonces estas traduciendo sin mucho conocimiento de Inglés?

**Comment by hiperiondev** [comment]
Tengo conocimiento de lectura tecnica, pero aquí hay varias cosas que me exceden.
El portugues lo leo bastante bien. Incluiré esa revisión tambien.

Muchas gracias

**Review by NikoVP — APPROVED** [review]

**Comment by AJenbo** [comment]
@NikoVP I see that you approved the changes, but it does not appear that @hiperiondev has applied your suggestions? Normally you do not approve before the requested changes have been made or you guys agreed to handle it differently.

Let me know if I should merge as is, apply the suggested changes (Preferably @hiperiondev would accept them so that we all agree) and merge, or something else.

**Comment by NikoVP** [comment]
Don´t know what hiperiondev plan is but probably the whole translation will need additional passes to get it to better shape. His corrections now improve some parts of it so its okay to commit. I'll refrain from correcting myself since its not my first or second language, Hopefully someone native will join in and do some more editing until 1.3.0 release comes.

**Comment by hiperiondev** [comment]
I'm not sure whether to add these suggestions in this commit or add them to the next one, maybe the latter is better and disregard this one.
Today I plan to finish the texts of the conversations and go over everything again. Possibly at night (here UTC -3) I could send the corrected ending.
In the translation I added many comments about parts that I am not sure or do not know how to translate.
After this it would be important that others join in to provide corrections so I am going to Discord with the hope that some charitable soul wants to collaborate.

**Comment by hiperiondev** [comment]
As I have told @NikoVP my second language is Italian so I plan to compare the translations that I do not consider correct against this translation. I can also use the Portuguese translation as I read it very well.
I think it can be a good alternative while we wait for someone else to join the project.

**Comment by hiperiondev** [comment]
For now, and being the only translator available, my plan is to release the corrections in steps and return to the beginning in different passes to refine the corrections.
In this way, the translation will be improved incrementally so that players can use it and find context errors. Hopefully we will start to get some very important feedback to continue.

**Comment by hiperiondev** [comment]
It would be interesting if the game itself had some indication on how to report translation errors so that players who do not know about programming know where to report them

**Comment by AJenbo** [comment]
If you agree with the suggestions you can simply add them by going to the file changes tab and click "Add suggestion to batch", and then commit them to the pull request.

image

You can also commit individual suggestion directly here in the overview, or write reply if you do not agree with the suggestion.
That is the preferred way to handle pull requests.

**Comment by AJenbo** [comment]
@inukaze has also been working on a Spanish translation, maybe he would be willing to help. He is also on Discord.

**Comment by hiperiondev** [comment]
I've added @NikoVP's suggestions to the next commit.
I will close this one so as not to create conflicts.

**Comment by hiperiondev** [comment]
It is continued in the next commit

**Comment by AJenbo** [comment]
Ok, you actually do not need to close the PR to update it you simply push your changes and the PR will update :)

**Comment by inukaze** [comment]
> @inukaze has also been working on a Spanish translation, maybe he would be willing to help. He is also on Discord.

Thanks, well i don't know the notification just arrive today. but yes, i had my own spanish translation finished three months ago. i just waiting the next windows release to put the spanish translation for give a copy to my friends to play the game with my spanish text translation.

**Comment by AJenbo** [comment]
You don't want to have it included in the release?

**Comment by inukaze** [comment]
@Ajenbo : i had strange bug if i translate "Resist Magic" (ID : 1316) to "Rest. Magica" / "Magica" / "Magicka" the Devilution Client or hang up or make a segmentation fault. 

The most accurate translation of "Resist Magic" to spanish is "Resistencia Magica" but is too long, because of that i use the abreviature "Rest." but the error looks like is something character quantity, because if i add "Rest. Magia1" i got same error. and is just adding one character more. 

Well i should leave the translation with "Rest. Magia" is not right but the spanish speakers goes to think "Resistencia a la magia".

**Pull request closed without merging**